tools/create_artifacts.sh

```diff
@@ -37,6 +37,7 @@ ANK_BIN_DIR="${DIST_DIR}/bin"
 RELEASE_FILE_NAME_BASE="ankaios-${RELEASE_ARCHITECTURE}"
 
 echo "Creating the archive for '$RELEASE_FILE_NAME_BASE'"
+chmod +x "${ANK_BIN_DIR}"/ank{,-server,-agent}
```

Comment:
The contents of this directory will finally be copied to /usr/local/bin so all files shall be executable. Why not just

```
chmod +x "${ANK_BIN_DIR}/*"
```

Reply:
Due to security reasons we shall not chmod blind everything that is inside a folder with a wildcard expression. This was also discussed in the meeting yesterday.

Reply:
OK for me.